docs/modules/theme/pages/fallback-font.adoc

```diff
@@ -95,3 +95,28 @@ font:
 ----
 
 Now Asciidoctor PDF will look for the emoji in the Symbola font instead of the Noto Emoji font.
+
+== Default Microsoft Windows Font Limitations
+When using the fonts provided by default in Microsoft Windows 10, and possibly other versions, it is important to define a fallback font to provide full coverage of Asciidoctor language features.
+In particular, the https://docs.asciidoctor.org/asciidoc/latest/lists/checklist/[Checklist] feature makes use of the _Ballet Box_ ☐ (U+2610) and _Ballot Box with Check_ "☑" (U+2611) characters.
+Two fonts provided by Windows, Segoe UI Symbol (`seguisym.ttf`) and Segoe UI Emoji (`seguiemj.ttf`), do contain these characters.
```

Comment:
Since these font files are paths, I would prefer the syntax here as well:

```
[.path]_seguisym.ttf_
```

(The stylesheet doesn't include any dedicated styles for the path role at the moment, but it may in the future, at which point it will be able to identify a value that it is a path).

Reply:
Thanks for your questions.

As for the page name, I do not want to draw that much attention to using Windows fonts. Therefore, I would like to proceed with adding this new section at the end of the "Use Custom Fonts" page instead.

> I am weary of giving the impression that it's okay to manipulate and repackage fonts provided by Windows due to licensing issues...The scope of this commit is to provide instructions for users using the fonts provided by Windows by default, so that Asciidoctor-PDF files will match their existing OS programs look-and-feel...

As long as fallback fonts are configured that cover all required characters, no modification of the primary font is needed. So the advice would be to keep specifying fallback fonts until all the requirements are met.

Keep in mind that Asciidoctor PDF is not optimized for using Windows Fonts. The application provides a set of built-in fonts that cover all required characters. If a user decided to use their own fonts, they are on their own to find suitable fonts that meet the requirements. Assisting with that is not the responsibility or goal of this project.

> I want to side-step this issue just by referencing what is already installed on a Windows users system.

That does come with some consequences/responsibility as that is not, like I said, a goal of this project.

> I discovered that neither seguisym.ttf and seguiemj.tff contain '\ufeff - zero width no-break space'. Therefore, I need to provide an extra fallback font... what is that character used for?

It looks like this entry is outdated. I can't find any reference to it in the code and rendering seems to work fine without it. I'm going to comment it out as a requirement in the docs for now until we discover otherwise.

> I want to wait for feedback before making any more commits.

Let's not over-complicate this. If you make the changes I requested, I think it will be sufficient to merge.

Reply:
I kept it simple, and simply changed the heading in commit 48c098682b30b0afd6feafaa52614244d4f76c0f.